src/compressed_tensors/quantization/quant_args.py

```diff
@@ -353,9 +358,17 @@ def validate_model_after(model: "QuantizationArgs") -> "QuantizationArgs":
             # default to minmax for non-dynamic cases
             observer = "minmax"
 
+        if zp_dtype is None:
+            zp_dtype = model.pytorch_dtype()
+
+        if scale_dtype is None:
+            scale_dtype = torch.bfloat16
```

Comment:
I think we should use float32 here. In practice, I don't think the scale's dtype matters much, but it is the defacto standard to use float32 scales for kernels like [DeepGEMM](https://github.com/deepseek-ai/DeepGEMM)

Reply:
I’ve thought about it some more and I don’t think we should just be defaulting to any dtype that isn’t the model’s dtype so I am going to revert this.

vLLM sets all scale dtype as the model dtype’s unless overwritten. I think we want to follow the same